www/source/docs/index.html.slim

```diff
@@ -10,113 +10,63 @@ section.docs--hero
+  .row
+    .columns.medium-12
       = link_to 'Download and Install the Habitat CLI', '/docs/install-habitat', class: 'button cta'
+      = link_to 'View Project on GitHub', 'https://github.com/habitat-sh/habitat', class: 'button outline contrast'
 
 hr
 
-section.docs--quicklink
+section.docs--get-started
   .row
     .columns.medium-12
-      .docs--quicklink--content
-        h2 Quick Links
-  .row.docs--quicklink--cards
-    .docs--quicklink--card--wrapper.columns.medium-4
-      .docs--quicklink--card
-        .docs--quicklink--card-heading.heading.a
-          ul.no-bullet
-            li
-              a href="/docs/best-practices"
-                | Guide
-        .body-content
-          h4
-            a href="/tutorials/build-your-own/"
-              | Hab Your Web App Guide
-          p
-            | Let's learn how to use Habitat to build and
-            | manage web apps using built-in support for different frameworks and languages.
-
-    .docs--quicklink--card--wrapper.columns.medium-4
-      .docs--quicklink--card
-        .docs--quicklink--card-heading.heading.a
-          ul.no-bullet
-            li
-              a href="https://habitat.sh/docs"
-                | Documentation
-        .body-content
-          h4
-            a href="/docs/reference#handlebars-helpers"
-              | Lifecycle Hook Helpers
-          p
-            | All of the QoL Handlebars helpers for use in your hooks
-            | in one convenient place.
-
-    .docs--quicklink--card--wrapper.columns.medium-4
-      .docs--quicklink--card
-        .docs--quicklink--card-heading.heading.a
-          ul.no-bullet
-            li
-              a href="https://habitat.sh/blog"
-                | Blog
-        .body-content
-          h4
-            a href="/blog/2017/09/a-journey-with-habitat-on-amazon-ecs-part1"
-              | Hab + ECS
-          p
-            | Want to learn how to use habitat with Amazon's cloud-based
-            | scheduler? Look no further , we've got you covered.
+      .docs--get-started--content
+        h2 Getting Started
 
-  .row.docs--quicklink--cards
-    .docs--quicklink--card--wrapper.columns.medium-4
-      .docs--quicklink--card
-        .docs--quicklink--card-heading.heading.a
-          ul.no-bullet
-            li
-              a href="/docs/best-practices"
-                | Guide
-        .body-content
-          h4
-            a href="/docs/developing-packages"
-              | Plan Guide
-          p
-            | Get started with how to write Habitat plans
-            | here.
+  section.get-started--boxes
+      div.row
+        div.columns.medium-12
+          a.get-started--boxes--item.card href="/docs/reference/#infographics"
+            div.columns.medium-6
+              h3.h3 Habitat in Pictures
+              p
+                | Take a high-level look at the entire Habitat landscape and workflow using this set of diagrams
```

Comment:
I'd put a `.` at the end of this sentence, and the ones below (in the other blocks) as well.